src/stories/components/ChecAvatar.stories.mdx

```diff
@@ -70,7 +68,7 @@ import TextField from '../../components/TextField.vue';
       },
       props: {
         image: {
-          default: text('Image URL', 'http://lorempixel.com/400/400/people/'),
+          default: text('Image URL', 'https://www.fillmurray.com/400/400'),
```

Comment:
I can't see a license on this, we should use something that provides for open source repackaging

Reply:
Good point. However we do use fillmurray already in the UI lib before this PR. `lorempixel` has been down for a while. Do you want me to go through and find something suitable and replace it everywhere it's in use?

Reply:
At some stage yeah, that'd be good

Reply:
Do you mind if I follow up on that? It looks like FillMurray uses creative commons images, and I can't find any that are different. We're probably fine using these as they are under CC now - provided they're used for layout only.

Reply:
I will merge and follow up later. I'd like to unblock dashboard work that uses these features.

Reply:
Yeah sorry, that's what I meant by later. All good!